### ISSUE TITLE
Creating a new job seems to lock up the entire UI

### ISSUE THREAD

**Issue by aapis** [aapis/ClockWork#166]
Can be fixed by restarting the app but the UI becomes totally unresponsive.

**Comment by aapis** [comment]
```
2023-08-11 11:35:15.532773-0600 ClockWork.Beta[20166:603964] [error] error: Serious application error.  Exception was caught during Core Data change processing.  This is usually a bug within an observer of NSManagedObjectContextObjectsDidChangeNotification.  -[Project compare:]: unrecognized selector sent to instance 0x600001374460 with userInfo (null)
```

**Comment by aapis** [comment]
This is fixed as of `beta-144` (#179)
